src/core/hle/service/hid/hid.cpp

```diff
@@ -1949,6 +1949,15 @@ class HidSys final : public ServiceFramework<HidSys> {
 
         RegisterHandlers(functions);
     }
+
+private:
+    void ApplyNpadSystemCommonPolicy(Kernel::HLERequestContext& ctx) {
+        // We already do this for homebrew so we can just stub it out
+        LOG_WARNING(Service_HID, "called");
```

Comment:
```suggestion
        LOG_WARNING(Service_HID, "(STUBBED) called");
```

Or change it to a debug log if it's not significant that it's stubbed.

Reply:
I think keeping it stubbed for now as this should be changed in the future with a controller rework. The only reason we have the system policy setup is because older homebrew when launched directly relies on the system to already have this set